app/controllers/tasks_controller.rb

```diff
@@ -10,14 +10,26 @@ def create
     redirect_to list_path(@list)
   end
 
+  def show
+    @task = Task.find(params[:id])
+    @list = @task.list_id
+  end
+
   def new
     @list = List.find(params[:list_id])
     @task = Task.new
   end
 
   def update
     @task = Task.find(params[:id])
-    @task.save
+    @list = List.find(@task.list_id)
+    @task.update(task_params)
```

Comment:
Are we bothered about whether or not this task can update? Do you think it would be useful to check the return status and perhaps show a flash message?

```ruby
if @task.update(task_params)
  redirect_to ...
else
  flash[:error] = ...
  redirect_to ..
```

…but actually working?